packages/mirage-flow-rawlink/mirage-flow-rawlink.1.0.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "thomas@gazagnaire.org"
+authors: ["Thomas Gazagnaire"]
+license: "ISC"
+homepage: "https://github.com/mirage/mirage-flow-rawlink"
+dev-repo: "https://github.com/mirage/mirage-flow-rawlink.git"
+bug-reports: "https://github.com/mirage/mirage-flow-rawlink/issues"
+doc: "https://mirage.github.io/mirage-flow-rawlink/"
+
+build: [ "jbuilder" "build" "-p" name "-j" jobs ]
```

Comment:
not many things to substitute here as it is a library only package. Could add the command call if you think that's still useful.